### ISSUE TITLE
Editor block submit button in form

### ISSUE THREAD

**Issue by philippbuerger** [livewire/flux#749]
Hi, I'm having a form with a button (type submit) in a modal. When rendering an editor the submit button isn't working anymore.

Without the editor everything works fine.

**Comment by philippbuerger** [comment]
Hi @jeffchown sure:

```
<?php

use function Livewire\Volt\{state, mount, on, rules};

state([
    'text',
]);

rules([
    'text' => ['required'],
]);

$update = function () {
    $this->validate();

    Flux::modal('edit-participant')->close();

    Flux::toast(
        text: __('Participant updated'),
        variant: 'success',
    );
};

?>

<div>
    <flux:modal.trigger name="edit-participant">
        <flux:button>Open</flux:button>
    </flux:modal.trigger>

    <flux:modal
        name="edit-participant"
        class="max-w-2xl space-y-6 w-full"
        @close="cancel"
    >

        <form
            wire:submit.prevent="update"
            class="space-y-6"
        >
            <div>
                <flux:heading size="lg">{{ __('Edit participant') }}</flux:heading>
                <flux:subheading>{{ __('Edit existing participant') }}</flux:subheading>
            </div>

            <flux:editor wire:model="text" />

            <div class="flex">
                <flux:spacer/>

                <flux:button
                    class="ml-2"
                    variant="primary"
                    type="submit"
                >
                    {{ __('Update') }}
                </flux:button>
            </div>
        </form>
    </flux:modal>
</div>
```

**Comment by jeffchown** [comment]
Great @philippbuerger , thanks.

I was able to reproduce the issue. 
(NOTE: I did have to remove the `@close="cancel"` from your example, it was throwing a method not found error)

The only way I was able to get it to work is:
- as you said, to replace the editor - with something like `flux:textarea` 
- or, add `wire:click="update"` directly to the Update button

So it looks like something in the `flux:editor` code is maybe trapping the form's default `submit` action.

@calebporzio @joshhanley Any ideas?

**Comment by philippbuerger** [comment]
@jeffchown thank you. I removed the `@close="cancel"` from my example.

**Comment by colinmac17** [comment]
Oops I think I made a duplicate issue here just after this! I recorded a video and share my code there though which may be helpful.

https://github.com/livewire/flux/issues/750

**Comment by jeffchown** [comment]
@colinmac17 Definitely the same issue 👍

**Comment by calebporzio** [comment]
Found the problem. It's the <flux:editor.link /> component. I use a form element inside it and you can't nest forms otherwise browsers will do really weird things.

For this moment, you can use the docs as reference and render the editor without the link toolbar item, otherwise, I'll hopefully get a fix out soon (just need a minute to figure out the best alternative)

![Image](https://github.com/user-attachments/assets/c6284f06-6818-4a2e-bb84-d293c360d6d4)

Thanks for the report

**Comment by calebporzio** [comment]
This has been fixed and tagged: please upgrade to v1.0.28
